.github/workflows/conformance-aks.yaml

```diff
@@ -152,7 +152,7 @@ jobs:
 
       - name: Install Cilium CLI
         run: |
-          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.8.4/cilium-linux-amd64.tar.gz
+          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.8.5/cilium-linux-amd64.tar.gz
           sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/bin
```

Comment:
Not directly related to this PR but I have 2 questions/suggestions:

- Why is the checksum not being used to verify the tarball?
- If we added a `stable.txt` file to the cilium-cli repo similarly to what is done in the Hubble and Cilium repos, we wouldn't have to open PRs to update Cilium CLI every time a new release is tagged.

TLDR, I'm suggesting to do something like this:
```
export CILUM_CLI_VERSION=$(curl -s https://raw.githubusercontent.com/cilium/cilium-cli/master/stable.txt)
curl -L --remote-name-all https://github.com/cilium/cilium-cli/releases/download/$CILUM_CLI_VERSION/cilium-linux-amd64.tar.gz{,.sha256sum}
sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
rm cilium-linux-amd64.tar.gz{,.sha256sum}
```

Reply:
Good points. Let's change it in this PR already. I'll send a PR adding `stable.txt` to `cilium-cli` and will then change this PR accordingly.

Reply:
Cilium CLI PR add `stable.txt` (and documenting the release process): #434

Reply:
Updated this PR accordingly.